doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -86,7 +86,44 @@ It covers the test cases against image management operations.
 
 
 #### 4.3.4.3 Block Storage - Cinder
-It covers the test cases against volume management operations.
+
+Cinder API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) and
+[cinder-tempest-plugin](https://opendev.org/openstack/cinder-tempest-plugin)
+as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
```

Comment:
Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`